nautobot_circuit_maintenance/handle_notifications/sources.py

```diff
@@ -251,13 +284,15 @@ def get_notifications(  # pylint: disable=too-many-locals, too-many-branches
                 logger.log_warning(message=value_error)
                 continue
 
-            rawnotification = imap_conn.receive_notifications(logger, restrict_emails, since)
-            received_notifications.extend(rawnotification)
+            if source.validate_providers(logger, notification_source, since_txt):
+                rawnotification = source.receive_notifications(logger, since)
+                received_notifications.extend(rawnotification)
```

Comment:
`rawnotification` is a list, not a single item, correct?
```suggestion
                raw_notifications = source.receive_notifications(logger, since)
                received_notifications.extend(raw_notifications)
```